src/index.html

```diff
@@ -12,6 +12,7 @@
       href="./styles/main.scss"
     />
```

Comment:
The `href` attribute in the link tag points to a `.scss` file. Browsers cannot directly interpret Sass files, so you need to compile this file to a `.css` file and link to that instead.